test/instrumentation/span.js

```diff
@@ -137,19 +135,16 @@ test('#_encode() - ended named', function myTest2 (t) {
   })
 })
 
-test('#_encode() - truncated', function myTest3 (t) {
-  var trans = new Transaction(agent)
+test('#_encode() - disabled stack traces', function (t) {
+  var ins = mockInstrumentation(function () {})
+  ins._agent._conf.captureSpanStackTraces = false
+  var trans = new Transaction(ins._agent)
   var span = new Span(trans)
-  span.start('foo', 'bar')
-  span.truncate()
+  span.start()
+  span.end()
   span._encode(function (err, payload) {
     t.error(err)
-    t.deepEqual(Object.keys(payload), ['name', 'type', 'start', 'duration', 'stacktrace'])
-    t.equal(payload.name, 'foo')
-    t.equal(payload.type, 'bar.truncated')
-    t.ok(payload.start > 0)
-    t.ok(payload.duration > 0)
-    assert.stacktrace(t, 'myTest3', __filename, payload.stacktrace, agent)
+    t.deepEqual(Object.keys(payload), ['transactionId', 'timestamp', 'name', 'type', 'start', 'duration'])
```

Comment:
You don't think we need to validate the contents of the object here?

Reply:
Fixed